pennylane/io.py

```diff
@@ -77,14 +78,24 @@ def load(quantum_circuit_object, format: str, **load_kwargs):
             Currently, only supported for Qiskit's `QuantumCircuit <https://docs.pennylane.ai/projects/qiskit>`_.
 
     Returns:
-        function: the PennyLane template created from the quantum circuit
-        object
+        function: the PennyLane template created from the quantum circuit object
+
+    .. warning::
+        ``qml.load`` is deprecated. Instead, please use the functions outlined in the
+        `Importing workflows <https://docs.pennylane.ai/en/stable/introduction/importing_workflows.html>`_
```

Comment:
This can be a dynamic link rather than static.

Reply:
How does dynamic links work?